### ISSUE TITLE
json file has errors/issues

### ISSUE THREAD

**Issue by tessus** [DNSCrypt/dnscrypt-resolvers#627]
The json file at https://download.dnscrypt.info/resolvers-list/json/public-resolvers.json has issues.

How is it possible to have an entry without an `addrs` key? This does not even make sense.

**Comment by ianbashford** [comment]
I see it's all of the odoh entries:

```
curl https://download.dnscrypt.info/resolvers-list/json/public-resolvers.json   |jq -r '.[] |  select(.addrs == null) |.name '
odoh-koki-es
odoh-jp.tiarap.org
odoh-resolver4.dns.openinternet.io
odoh-cloudflare
odoh-koki-ams
odoh-tiarap.org
odoh-jp.tiar.app
odoh-koki-se
odoh-id-gmail
odoh-crypto-sx
```

@jedisct1  do you have a script that generates the json ?  I couldn't find anything at a glance...

**Comment by jedisct1** [comment]
@ianbashford it's the output of the `dnscrypt-proxy -list-all -json` command.

**Comment by jedisct1** [comment]
And it should be fixed in 4c67e790f68025ca6822ff1f29002d9166fb5008 ;)
